drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -12,6 +12,14 @@ config WIFI_NRF700X
 	select NET_L2_ETHERNET_MGMT if NETWORKING && NET_L2_ETHERNET
 	select WIFI_USE_NATIVE_NETWORKING if NETWORKING
 	select EXPERIMENTAL if !SOC_SERIES_NRF53X && !SOC_SERIES_NRF91X
+	# For relocation code to external flash
```

Comment:
If not already done, I would suggest to add another Kconfig, like CONFIG_WIFI_PATCHES_EXT_FLASH, which would select all dependent configs. We would then decide in the application code weather or not we want to offload the internal flash by default or just in some specific configurations.

Reply:
Is there a downside in always using external flash? If not, we can make this default, else, I can protect this with a Kconfig.

Reply:
Well, probably most of (I dare to say all) Matter customers are including external flash in their end products, but I guess we cannot assume this is always the case, especially for smaller applications, leveraging from our WiFi implementation.

Reply:
Sure, but I have enabled this feature only for nRF5340, see the custom linker script, so, it won't be applicable for anything else. Anyways, `CUSTOM_LINKER_SCRIPT` will always be set even if this feature isn't supported e.g., on nRF7002 which is undesirable, added a Kconfig.